anaconda/Dockerfile

```diff
@@ -1,32 +1,74 @@
-FROM debian:latest
-
-#  $ docker build . -t continuumio/anaconda:latest -t continuumio/anaconda:2018.12 -t continuumio/anaconda2:latest -t continuumio/anaconda2:2018.12
-#  $ docker run --rm -it continuumio/anaconda2:latest /bin/bash
-#  $ docker push continuumio/anaconda:latest
-#  $ docker push continuumio/anaconda:2018.12
-#  $ docker push continuumio/anaconda2:latest
-#  $ docker push continuumio/anaconda2:2018.12
+FROM alpine:3.9 as alpine-glibc
 
+LABEL MAINTAINER="Vlad Frolov"
```

Comment:
What's this?

Reply:
He's the maintainer of the Dockerfile that adds glibc to alpine, or are you asking about labels?